homeassistant/components/bluesound/media_player.py

```diff
@@ -334,110 +327,49 @@ async def async_update(self) -> None:
         if not self._is_online:
             return
 
-        await self.async_update_sync_status()
-        await self.async_update_presets()
-        await self.async_update_captures()
-        await self.async_update_services()
-
-    async def send_bluesound_command(
-        self, method, raise_timeout=False, allow_offline=False
-    ):
-        """Send command to the player."""
-        if not self._is_online and not allow_offline:
-            return
-
-        if method[0] == "/":
-            method = method[1:]
-        url = f"http://{self.host}:{self.port}/{method}"
-
-        _LOGGER.debug("Calling URL: %s", url)
-        response = None
-
-        try:
-            websession = async_get_clientsession(self._hass)
-            async with timeout(10):
-                response = await websession.get(url)
-
-            if response.status == HTTPStatus.OK:
-                result = await response.text()
-                if result:
-                    data = xmltodict.parse(result)
-                else:
-                    data = None
-            elif response.status == 595:
-                _LOGGER.info("Status 595 returned, treating as timeout")
-                raise BluesoundPlayer._TimeoutException
-            else:
-                _LOGGER.error("Error %s on %s", response.status, url)
-                return None
-
-        except (TimeoutError, aiohttp.ClientError):
-            if raise_timeout:
-                _LOGGER.info("Timeout: %s:%s", self.host, self.port)
-                raise
-            _LOGGER.debug("Failed communicating: %s:%s", self.host, self.port)
-            return None
-
-        return data
+        with suppress(TimeoutError):
+            await self.async_update_sync_status()
+            await self.async_update_presets()
+            await self.async_update_captures()
```

Comment:
Why do we suppress these now?

Reply:
`send_bluesound_command`(which i replaced by the library) was called with `raise_timeout=False` which suppressed `TimeoutError`. I pulled that logic one level up and put it here.